### PR TITLE
Add Bin

### PR DESCRIPTION
**Pull request by junosuarez**
this lets you do fun things like
```sh
> npm i -g covfefe
...
> covfefe foo
foo covfefe
```
or even more handy, using [npx](https://github.com/zkat/npx):
```sh
> npx covfefe foo
...
foo covfefe
```

this is very important for reasons

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,8 +1,11 @@
 {
   "name": "covfefe",
-  "version": "1.1.3",
+  "version": "1.1.2",
```
Comment (rgbkrk) [inline package.json]: This should stay the same and not change here, I'll fix it up before shipping.

**Review by rgbkrk — APPROVED** [review]
despite the negative feedback covfefe

**Comment by rgbkrk** [comment]
Thanks for the handy bin tool!

**Comment by rgbkrk** [comment]
Whoops, shipped that as patch rather than a minor release.
¯\\\_(ツ)\_/¯

File: package.json (hunk 1 above)
Comment (junosuarez) [reply package.json]: oops, thanks for catching and fixing that!

**Comment by KoryNunn** [comment]
That's ok we can easily resolve any issues that may arise from this by covfefe

**Pull request merged**